larpandoracontent/LArHelpers/LArFormattingHelper.h

```diff
@@ -0,0 +1,222 @@
+     */
+    static void ResetStyle(std::ostream &stream=std::cout);
+
+    /**
+     *  @brief  Reset the text colour of the standard output stream
+     */
+    static void ResetColour(std::ostream &stream=std::cout);
+
+    /**
+     *  @brief  Reset the formatting and text colour of the standard output stream
+     */
+    static void Reset(std::ostream &stream=std::cout);
+
+    /**
+     *  @brief  Print a formatting character to the standard output stream
+     *
+     *  @param  code the formatting code to output
+     */
+    static void PrintFormatCharacter(const unsigned int &code, std::ostream &stream=std::cout);
+
+    /**
+     *  @brief  Get a formatting character
+     *
+     *  @param  code the formatting code to output
+     */
+    static std::string GetFormatCharacter(const unsigned int &code);
+
+    /**
+     *  @brief  Print a header line of a given width
+     *
+     *  @param  title the title of the header
+     *  @param  width the width of the header line
+     */
+    static void PrintHeader(const std::string &title = "", const unsigned int &width = 140);
+
+    /**
+     *  @brief  Print a horizontal rule
+     *
+     *  @param  width the width of the rule line
+     */
+    static void PrintRule(const unsigned int &width = 140);
+
+    enum Style : int
+    {
+        REGULAR = 0,
+        BOLD = 1,
+        DIM = 2,
+        UNDERLINED = 4,
+        INVERTED = 7
+    };
+    
+    enum Colour : int
+    {
+        DEFAULT = 39,
+        BLACK = 30,
+        RED = 31,
+        GREEN = 32,
+        YELLOW = 33,
+        BLUE = 34,
+        MAGENTA = 35,
+        CYAN = 36,
+        LIGHT_GRAY = 37,
+        DARK_GRAY = 90,
+        LIGHT_RED = 91,
+        LIGHT_GREEN = 92,
+        LIGHT_YELLOW = 93,
+        LIGHT_BLUE = 94,
+        LIGHT_MAGENTA = 95,
+        LIGHT_CYAN = 96,
+        WHITE = 97
+    };
+
+    //--------------------------------------------------------------------------------------------------------------------------------------
+
+    class Table 
+    {
+    public:
+
+        /**
+         * @brief  Table constructor
+         *
+         * @param  columnTitles the string columns titles use empty string for a separator column
+         * @param  precision the number of significant figures to display for number type table elements
+         */
+        Table(const std::vector<std::string> &columnTitles, const unsigned int &precision=3);
+
+        /**
+         * @brief  Add an element to the table into the next (non-separator) column
+         *
+         * @param  value the element to add to the table
+         * @param  style the style of the element
+         * @param  colour the colour of the element
+         */
+        template<class T>
+        void AddElement(const T &value, const Style &style=REGULAR, const Colour &colour=DEFAULT);
+
+        /**
+         * @brief  Print the table
+         */
+        void Print();
+    private:
```

Comment:
Add blank line here, between Print(); and private:

Can Print be a const member function?